capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -6,28 +6,55 @@ import { act } from "react-dom/test-utils";
 import BookSearchTile from "./components/BookSearchTile.jsx";
 import App from "./App.jsx";
 import {
-  Route, 
-  BrowserRouter as Router, 
+  Route,
+  BrowserRouter as Router,
   Link
-} from 'react-router-dom'; 
+} from 'react-router-dom';
 
 const container = document.createElement("div");
 document.body.appendChild(container);
 const book = {
   title: "Title",
-  author: ["Author"],
+  authors: ["Author"],
   thumbnailLink: "https://http.cat/100.jpg"
 };
+const getUserBookLists = () => {
+  let userBookLists = [{ id: "1", name: "Best Books" },
+  { id: "2", name: "Fantasy" },
+  { id: "3", name: "Sci-fi" }];
+  return userBookLists;
+}
+
+const getUserClubs = () => {
+  let userClubs = [{ id: "1", name: "Cool Kats Book Club" },
+  { id: "2", name: "Sci-fi Fans" }];
+  return userClubs;
+}
+
+const getUserCommunities = () => {
+  let userCommunities = [{ id: "1", name: "Sci-fi Community" },
+  { id: "2", name: "Read-a-lot" }];
+  return userCommunities;
+}
 
 test('renders book search tile test', () => {
-  render(<BookSearchTile title={book.title} author={book.author} thumbnailLink={book.thumbnailLink} />, container);
+  render(<BookSearchTile book={book}
```

Comment:
I like the way you turned the book into a json object instead of having several different props to pass!